code/modules/organs/subtypes/replicant.dm

```diff
@@ -0,0 +1,171 @@
+	desc = "A juicy piece of.. rubber, found in someone's head?"
+	can_reject = FALSE
+	icon_state = "brain_grey"
+
+/obj/item/organ/internal/brain/replicant/torso
+	parent_organ = BP_TORSO
+
+/obj/item/organ/internal/voicebox/replicant
+	name = "replicant voicebox"
+	desc = "A rubbery piece of meat used for vocalizations."
+	can_reject = FALSE
+
+/obj/item/organ/internal/heart/replicant
+	name = "replicant heart"
+	desc = "A mass of rubber and muscle used for pumping fluid."
+	can_reject = FALSE
+	icon_state = "heart_grey-on"
+	dead_icon = "heart_grey-off"
+
+/obj/item/organ/internal/lungs/replicant
+	name = "replicant lungs"
+	desc = "A pair of rubbery sacs used for respiration."
+	can_reject = FALSE
+	icon_state = "lungs_grey"
+
+/obj/item/organ/internal/liver/replicant
+	name = "replicant liver"
+	desc = "A mass of rubber used for filtering and breaking down chemicals."
+	can_reject = FALSE
+	icon_state = "liver_grey"
+
+/obj/item/organ/internal/kidneys/replicant
+	name = "replicant kidneys"
+	desc = "A pair of small sacs used for filtering chemicals."
+	can_reject = FALSE
+	icon_state = "kidneys_grey"
+
+/obj/item/organ/internal/xenos/plasmavessel/replicant
+	name = "replicant phorogenic sac"
+	desc = "A bulbous rubbery mass that converts nutrients from the host into a biological compound eerily similar to phoron."
+	can_reject = FALSE
+	icon_state = "plasma_grey"
+
+/obj/item/organ/internal/xenos/acidgland/replicant
+	name = "replicant aerosol tubule"
+	desc = "A long, rubbery tube that ends in a hard plastic-like bulb."
+	can_reject = FALSE
+	icon_state = "acidgland_grey"
+
+/obj/item/organ/internal/xenos/resinspinner/replicant
+	name = "replicant biomesh spinner"
+	desc = "A rubbery mass with protrusions for molding organic material."
+	can_reject = FALSE
+	icon_state = "xenode_grey"
+
+/*
+ * These are unique organs to Replicants and other Ancient Aliens, though they can be used elsewhere. They follow the same rules.
+ */
+
+/obj/item/organ/internal/immunehub
+	name = "lymphomatic control web"
+	desc = "A mesh of twitching strings."
+	organ_tag = O_AREJECT
+	parent_organ = BP_TORSO
+	icon_state = "immunehub"
+
+	var/rejection_adjust = 10
+
+/obj/item/organ/internal/immunehub/replicant
+	name = "replicant assimilation web"
+	desc = "A mesh of jiggling rubber strings that dig at nearby flesh."
+	can_reject = FALSE
+
+/obj/item/organ/internal/immunehub/handle_organ_proc_special()
+	if(!owner)
+		return
+
+	var/list/all_organs = list()
+	all_organs |= owner.internal_organs
+	all_organs |= owner.organs
+
+	var/modifier = round(rejection_adjust * (1 - 0.5 * is_bruised()))
+
+	for(var/obj/item/organ/I in all_organs)
+		I.rejecting = max(0, rejecting - modifier)
+
+/obj/item/organ/internal/metamorphgland
+	name = "morphoplastic node"
+	desc = "A strange clump of meat that doesn't quite stay in place."
+	organ_tag = O_VENTC
+	parent_organ = BP_GROIN
+	icon_state = "innards"
+	organ_verbs = list(
+		/mob/living/proc/ventcrawl
```

Comment:
Did you test this? If so, what were the results? I'm immensely curious.

Reply:
The person can ventcrawl with the organ, though it doesn't allow them to crawl with things like clothing or other items still.